docs/index.html

```diff
@@ -138,7 +106,15 @@ <h1>Other attributes</h1>
 			</tr>
 		</thead>
 		<tbody>
-			<tr>
+			<tr property="other-attribute" mv-multiple>
```

Comment:
Avoid hyphens in properties, it means we can't use them in expressions (this would be interpreted as other minus attribute in an expression!)

Btw since you already have data, `mv-alias` will prove useful to preserve it: https://mavo.io/docs/migration